parquet-column/src/test/java/org/apache/parquet/column/statistics/TestStatistics.java

```diff
@@ -384,6 +384,23 @@ public void testBinaryMinMax() {
   }
 
   @Test
+  public void testBinaryMinMaxForReusedBackingByteArray() {
+    BinaryStatistics stats = new BinaryStatistics();
+
+    byte[] bytes = new byte[] { 10 };
+    stats.updateStats(Binary.fromReusedByteArray(bytes));
```

Comment:
You should be able to create the Binary just once, change the backing bytes, and use it in each call to `updateStats`.